cpp/daal/src/algorithms/dtrees/dtrees_model.cpp

```diff
@@ -38,6 +38,39 @@ Tree::~Tree() {}
 
 ModelImpl::ModelImpl() : _nTree(0) {}
 
+ModelImpl::ModelImpl(const ModelImpl & other)
```

Comment:
Perhaps some small tests could be added for the ones with non-default constructors.